### ISSUE TITLE
Update Wiki

### ISSUE THREAD

**Issue by Dayaaf** [GTLugo/ExtraGunGear#2]
Umm.. The wiki isn't updated i dont think. I was playing with this mod and i put my musket in for the guide, and there was a golden sniper rifle, which isn't in the wiki. There are more items, so if you could please update the wiki

**Comment by GTLugo** [comment]
Yeah, I'm in the process of adding a lot of stuff to the mod.  I just ask that you be patient, as I hope the wait will be well worth it. :)

**Comment by Ayel91** [comment]
I can help!
